org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/XtextEditor.java

```diff
@@ -593,12 +593,12 @@ public void selectionChanged(final SelectionChangedEvent event) {
 		}
 	}
 
-	private void installHighlightingHelper() {
+	protected void installHighlightingHelper() {
```

Comment:
Please add `@since 2.38` tags to the newly exposed methods

Reply:
Done